main.tf

```diff
@@ -130,15 +131,16 @@ locals {
 }
 
 module "gke_app" {
-  source  = "wandb/wandb/kubernetes"
-  version = "1.4.1"
+  # source  = "wandb/wandb/kubernetes"
+  # version = "1.4.1"
+  source = "../terraform-kubernetes-wandb"
```

Comment:
This is due to the local test I'm performing with this PR wandb/terraform-kubernetes-wandb#23